src/main/main.c

```diff
@@ -234,7 +234,7 @@ void init(void)
     servoMixerInit(masterConfig.customServoMixer);
 #endif
 
-#ifdef SPEKTRUM_BIND
+#if defined(SERIAL_RX) && defined(SPEKTRUM_BIND)
```

Comment:
Ideally, yes. But in reality there must be a possibility to both select and de-select a feature/function. There are many reasons why you may want to de-select an otherwise default item, lack of space is one. Do not know why this target has de-selected serial-RX. 

But OK, the target.h should be responsible for all dependencies too, as we do not have any automatic way of doing that. This would minimize conditional complexity in the common source code. Better to have a lot of special specifics in one target.h, than spreading complexity all over the place. Just for support of one single target. Currently one target, that is! Things may/will change... but lets deal with that then.